datahub-web-react/src/app/entity/mlFeatureTable/profile/features/TableOfMlFeatures.tsx

```diff
@@ -40,6 +40,7 @@ export default function TableOfMlFeatures({ features }: Props) {
     const entityRegistry = useEntityRegistry();
 
     const [tagHoveredIndex, setTagHoveredIndex] = useState<string | undefined>(undefined);
+    const [expandedRows, setExpandedRows] = useState({});
```

Comment:
minor: could you add a type signature here for type safety? `useState<{ [key: number]: boolean }>({});`